src/snapshotter.test.js

```diff
@@ -237,10 +241,16 @@ describe('The snapshotter', () => {
       onError
     );
     await mockSnapshot.takeSnap();
-    expect(seleniumMock).toBeCalledWith(mockSnapshot.driver, By);
+
+    expect(executeScriptMock).toBeCalledTimes(1);
+    expect(executeScriptMock.mock.calls[0][0]).toBeInstanceOf(Object);
```

Comment:
not sure how much value this provides. If its the selenium object I think we might able to use `mockSnapshot.driver, By`